contracts/aggregator/mainnet/flashloan/main.sol

```diff
@@ -209,7 +204,17 @@ contract FlashAggregator is Setups {
                 InstaFlashReceiverInterface(sender_).executeOperation(tokens_, amounts_, instaLoanVariables_._instaFees, sender_, data_);
             }
 
+            if (tokens_[0] == stEth) {
+                wstEth.wrap(amounts_[0]);
```

Comment:
I think we might need to wrap here a little more? Maybe 2 wei something. Else having wstETH balance on our contract will also work I think.

Reply:
So, the tests were passing always, and once we generate revenue we have logics for in the transferFeeToTreasury function to keep some amount in the contract.

Reply:
Got it. So ignoring this as of now then.